crates/api_crud/src/site/create.rs

```diff
@@ -552,6 +563,7 @@ mod tests {
     site_is_federated: Option<bool>,
     site_application_question: Option<String>,
     site_registration_mode: Option<RegistrationMode>,
+    site_oauth_registration: Option<bool>,
```

Comment:
Seems like this is always None, so you can remove the param and hardcode to None below.